openvpn/management.go

```diff
@@ -26,14 +26,15 @@ type ManagementMiddleware interface {
 	Start(connection net.Conn) error
 	Stop() error
 	ConsumeLine(line string) (consumed bool, err error)
+	State() (state State)
```

Comment:
Interface is collapsing

Reply:
You need State() only in tests. So construct exact structure (not interface) there